core/templates/pages/contact-page/contact-page.component.html

```diff
@@ -122,36 +122,29 @@ <h2 [innerHTML]="'I18N_CONTACT_PAGE_OTHER_INQUIRIES_HEADING' | translate"></h2>
   }
 
   :host {
-    .oppia-contact-page-content button {
+    .oppia-contact-page-content .contact-page-button {
```

Comment:
Shouldn't these classes be passed through the button's custom classes? The outer component should not know *anything* about the internal structure of the button.

Reply:
I'm not quite sure what you mean, we are passing a custom class and using it. Do you mind clarifying further?

Reply:
Sorry, I think I was reading the old thing -- this is fine. Please ignore this comment (apologies).